src/commands/app/create.ts

```diff
@@ -96,7 +97,15 @@ export default class Create extends BaseCommand<typeof Create> {
           message: this.messages.CONFIRM_CLONE_BOILERPLATE,
         }))
       ) {
-        await this.boilerplateFlow();
+        const boilerplate = await selectedBoilerplate();
```

Comment:
@harshithad0703 , You are attempting to use boilerplate directly below, and the boilerplate return type is any.The entire process will break if the boilerplate is an empty object.

Reply:
added conditions on if